.github/workflows/test-environment.yml

```diff
@@ -322,10 +323,8 @@ jobs:
       - name: Deploy CSPM GCP agent
         id: cspm-gcp-agent
         working-directory: deploy/deployment-manager
-        env:
-          DEPLOYMENT_LABELS: ${{ env.GCP_DEFAULT_TAGS }}
         run: |
-          . ./set_env.sh && ./deploy.sh
+          . ./set_env.sh && ./deploy.sh && gcloud deployment-manager deployments update "${DEPLOYMENT_NAME}" --update-labels "${GCP_DEFAULT_TAGS}"
```

Comment:
Initially, the labeling was introduced as part of the deploy.sh script, but it was decided to [revert](https://github.com/elastic/cloudbeat/pull/2114) this to avoid interfering with the customer's label policy. The new implementation runs the deploy.sh script first and then updates the deployment labels.